docs/content/en/docs/clustermgmt/etcd-backup-restore/ubuntu-rhel-etcd-backup.md

```diff
@@ -40,7 +41,10 @@ chown ec2-user snapshot.db
 scp -i $PRIV_KEY ec2-user@$ETCD_VM_IP:/home/ec2-user/snapshot.db . 
 ```
 
-NOTE: This snapshot file contains all information stored in the cluster, so make sure you save it securely (encrypt it).
+NOTE: 
```

Comment:
Nit but I'd change these to bullets instead of numbered list, we typically use numbered list for procedural steps.